src/Microsoft.Kusto.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -802,7 +803,13 @@ public ListDatabasesResponse ListDatabases(ListDatabasesParams listDatabasesPara
             }
 
             IEnumerable<DataSourceObjectMetadata> databaseMetadata = dataSource.GetChildObjects(objectMetadata);
-            if(databaseMetadata != null) response.DatabaseNames = databaseMetadata.Select(objMeta => objMeta.PrettyName).ToArray();
+            if (databaseMetadata != null)
+            {
+                response.DatabaseNames = databaseMetadata
+                    .Select(objMeta => objMeta.PrettyName == objMeta.Name ? objMeta.PrettyName : $"{objMeta.PrettyName} ({objMeta.Name})")
+                    .ToArray();
+            }
+
```

Comment:
Changed the way we render the database name when the names don't match. If they're the same then it'll render `DatabaseName` as usual but if they're different then it'll render as `Pretty Name (DatabaseName)` because we need the real name to query.